microshift_release_notes/microshift-4-19-release-notes.adoc

```diff
@@ -60,8 +60,10 @@ With this release, configurable transport layer security (TLS) protocols on inte
 [id="microshift-4-19-support_{context}"]
 === Support
 
-//[id="microshift-4-19-example-support-feature_{context}"]
-//==== Example Support feature here
+[id="microshift-4-19-telemetry_{context}"]
+==== Telemetry now available
+With this release, {microshift-short} adds the Telemeter API. Lightweight attributes can be sent from a connected cluster to Red{nbsp}Hat to monitor the health of the cluster. To opt-out of Telemetry, see
+//xref:../microshift_support/microshift-remote-cluster-monitoring.adpc#microshift-remote-cluster-monitoring[Remote health monitoring with a connected cluster].
```

Comment:
I was going to ask if linking to a nonexistent file would break the build even if it's commented out, but I'm guessing that's why it's a ".adpc" file for now? Just checking (and pretty neat workaround if so)

Reply:
That's a typo in the filename, lol. But yeah, Aidan and I talked about the bot several months ago and the builds now work if you comment out the non-existent xref.